.gitignore

```diff
@@ -1,5 +1,7 @@
-# Build directory
+# Build files
 /build
+/crossdock/crossdock
+/crossdock/jaeger-docker-compose.yml
```

Comment:
this a temporary thing? don't think we should be ignoring this completely

Reply:
crossdock/crossdock is the executable binary itself. Actually changed this in my recent crossdock development. The `jaeger-docker-compose.yml` is a downloaded file to launch the other jaeger clients.